README.md

```diff
@@ -10,53 +10,72 @@ https://molotow11.github.io/JsSimpleCart/demo/
 	2. Configure this part for link it to products:
 ```javascript
 const settings = {
-	productSelectors: {
-		product: ".product", // add class to each product on your site *
-		buttonAddToCart: ".product__add-to-cart-button", // add class to each button in your product *
-		title: ".product__title", // add class to each title in your product *
-		price: ".product__price", // add class to each price in your product *
-		qty: ".product__qty",
-		currency: ".product__currency", // add class to each currency in your product *
-	},
-	cartSelectors: {
-		body: "SimpleCart",
-		details: "SimpleCartDetails",
-		detailsClose: ".SimpleCartDetails__close",
-		detalisHidden: "SimpleCartDetails--hidden",
-		productAdded: "SimpleCart--product-added",
-		productNotAdded: "SimpleCart--product-not-added",
-	},
-	currencies: {
-		parentSelector: "products__currencies", // add id to place in your site a currencies select
-		BASE: "USD",  // set base currency of products *
-		list: [
-			{
-				BASE: "USD", // set one of two currencies *
-				symbol: "$", // set one of two currencies symbols *
-			},
-			{
-				BASE: "RUB", // set two of two currencies
-				symbol: "₽", // set two of two currencies symbols
-				rates: {
-					USD: 90, // set the exchange rate of the first currency to the second
-				},
-			},
-		],
-	},
-	cartCookieDays: 7, // set the time data saved to the user
-	formSubmitUrl: "/order-submit.php",
-	cookieProductsName: "SimpleCart",
+  productSelectors: {
+    product: ".product", // Add class to each product on your site. *
+    buttonAddToCart: ".product__add-to-cart-button", // Add class to each button in your product. *
+    title: ".product__title", // Add class to each title in your product. *
+    price: ".product__price", // Add class to each price in your product. *
+    qty: ".product__qty",
+    currency: ".product__currency", // Add class to each currency in your product. *
+  },
+  cartSelectors: {
+    body: "SimpleCart",
+    details: "SimpleCartDetails",
+    detailsClose: ".SimpleCartDetails__close",
+    detalisHidden: "SimpleCartDetails--hidden",
+    productAdded: "SimpleCart--product-added",
+    productNotAdded: "SimpleCart--product-not-added",
+  },
+  currencies: {
+    parentSelector: "products__currencies", // Add id to element in your site for to add currency select list.
+    code: "USD", // Set default currency of products. *
+    list: [ // You can add more currencies to the list or change the existing ones.
+      {
+        code: "USD", // Set currency code. *
+        symbol: "$", // Set currency symbol. *
+        rates: { // Set currency rate to all currencies in list.
```

Comment:
Please recode to use single value related to `base` currency

Reply:
So no need to get conversion rates for each currency

Reply:
Do you propose to use exchange rates against the dollar only?

Reply:
@BeastJsWeb Yes, only single exchange rate for each currency, not `rates` but something like `rateToBase`. 
So, not against dollar, but base currency. Please start thinking more abstractive.

Reply:
Ok, I think I see